### PR TITLE
CORS Update & New Config Params

### PR DESCRIPTION
**Pull request by JoeMcRobot**
Applications that have a separate front-end and back-end need to be able to update the CORS Origin, Allow Headers, and Allow Methods. We were running into various CORS errors on the front-end and need to be able to change these params.

The `cors.init_app` function has been updated with new params and the following config params have been added `CORS_ORIGIN`, `CORS_ALLOW_HEADERS`, and `CORS_ALLOW_METHODS` The admin docs have been updated as well.

### CODE REVIEW

File: lemur/factory.py [hunk 1]
```diff
@@ -160,14 +160,17 @@ def configure_extensions(app):
             environment=app.config.get("LEMUR_ENV", ''),
         )
 
-    if app.config["CORS"]:
-        app.config["CORS_HEADERS"] = "Content-Type"
+    if app.config.get("CORS"):
+        origins = app.config.get("CORS_ORIGIN", "*")
+        allow_headers = app.config.get("CORS_ALLOW_HEADERS", "Content-Type")
```
Comment (charhate) [inline lemur/factory.py:165]: Can you add `Authorization` to allowed header list, since lemur needs it.

**Comment by charhate** [comment]
@JoeMcRobot Thanks for contributing. I've left a comment, but otherwise it looks good.

**Comment by hosseinsh** [comment]
closing this PR due to inactivity, please feel free to open it as needed.

**Comment by hosseinsh** [comment]
hi @JoeMcRobot sorry, missed the follow-up here. can you please resubmit this PR. Thanks

**Pull request closed without merging**